### PR TITLE
Additions and adjustment of headwear for HoS/Warden + Minor addition to items in hos locker 

### PR DESCRIPTION
**Pull request by Aboshedab**
1- Black hos beret that matches the current hos items. The darker colored attire for the HoS did not have headwear that matched the color scheme.
2- HoS locker comes with handcuff and a crowbar now.

Updates:
3- Added black hat for HoS and warden.
4- Hats worn by HoS/Warden no longer act like helmets. 
5- Adjust worth values accordingly of said items.
6- Extremely minor tidy up, added a space between a comma and a word.
7- HoS and Warden now have selections of their headwear in the loadout.

Sprites provided by Stryker and Wowzewow.

Note 1: I originally did a lot more work than this like giving the Warden their own helmet to adjusting armor values due to inconsistencies to increase energy values on ablatives which were left untouched due to old code when weapons that check for energy were added and a bunch of other things. With the recommendations of people like Matt and Wezzy, I'll be breaking them up into several PR's so for the time being I will not be adding more to keep it as atomic as possible. If anything is undocumented here or the changelog, feel free to DM me on discord for a faster response.

Note 2: You can expect similar treatment done to regular attire like the dress jackets losing

### CODE REVIEW

**Comment by Aboshedab** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 1]
```diff
@@ -62,10 +62,15 @@
...
 /datum/gear/head/beret/hos
-	display_name = "beret, security (head of security)"
+	display_name = "blue beret, security (head of security)"
 	path = /obj/item/clothing/head/beret/sec/hos
 	allowed_roles = list("Head of Security")
 
+/datum/gear/head/beret/hos/alt
+	display_name = "black beret, security (head of security)"
+	path = /obj/item/clothing/head/beret/sec/hos/alt
+	allowed_roles = list("Head of Security")
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:72]: This should all be a single selection I think?

Comment (Aboshedab) [reply code/modules/client/preference_setup/loadout/loadout_head.dm:72]: Can be done.

Comment (Aboshedab) [reply code/modules/client/preference_setup/loadout/loadout_head.dm]: Done.

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 2]
```diff
@@ -63,9 +63,17 @@
 
 /datum/gear/head/beret/hos
 	display_name = "beret, security (head of security)"
+	description = "A selection of commander beret."
```
Comment (alsoandanswer) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:66]:
```suggestion
	description = "A selection of commander berets."
```

**Comment by Aboshedab** [comment]
Going to add some more stuff

**Comment by Aboshedab** [comment]
!wip

**Comment by Aboshedab** [comment]
!review

**Comment by Aboshedab** [comment]
Read the revised writing in the main post for the changes.

**Review by alsoandanswer — APPROVED** [review]

**Review by fernerr — APPROVED** [review]

**Pull request merged**